src/FSLibrary/MissionSorobanClassicMixed.fs

```diff
@@ -0,0 +1,82 @@
+// Copyright 2019 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module MissionSorobanClassicMixed
+
+open StellarCoreSet
+open StellarCorePeer
+open StellarMissionContext
+open StellarFormation
+open StellarStatefulSets
+open StellarSupercluster
+open StellarNetworkData
+open StellarCoreHTTP
+
+let sorobanClassicMixed (context: MissionContext) =
+
+    let context =
+        { context with
+              numAccounts = 2000
+              numTxs = 2000
+              txRate = 20
+              skipLowFeeTxs = true }
+
+    let sorobanCoreSet =
+        MakeLiveCoreSet
```

Comment:
need to configure quorum sets for both core sets, otherwise doesn't this run two forks?